server/routes/form.js

```diff
@@ -409,26 +409,31 @@ module.exports = function Index({
       const section = 'supervisor'
       const form = 'review'
       const formPageConfig = formConfig[section][form]
-
-      if (!formService.isValid(formPageConfig, req, res, section, form, bookingId)) {
-        return
+      if (req.body.supervisorOverriddenCategory !== 'D' && req.body.supervisorOverriddenCategory !== 'J') {
+        if (!formService.isValid(formPageConfig, req, res, section, form, bookingId)) {
```

Comment:
dont need to validate if open conditions input is required as we aren't persisting the supervisor form